lesson2/NewProject/lib/src/main/java/com/example/MoneySaver.java

```diff
@@ -22,11 +22,12 @@ public Account getMyAccount() {
     }
 
     public String toString() {
-        return name + ", ";
+        return name + ", " + "My balance is: "+ myMoney;
     }
 
     public void deposit(long amount) {
         myAccount.deposit(amount);
+        this.myMoney -= amount;
```

Comment:
For the correct implementation of the deposit function, subtract the money you put in your account since you don't have it in your pocket anymore.